src/sentry/sentry_metrics/use_case_id_registry.py

```diff
@@ -7,14 +7,16 @@
 
 
 class UseCaseID(Enum):
+    SPANS = "spans"
```

Comment:
Do we need spans in the UseCaseID enum right now? If not can we remove it until the new use case is being onboarded?

Reply:
Still like to keep it for testing the interaction between postgres indexer and writes limiter ([here](https://github.com/getsentry/sentry/pull/49138/files#diff-4ecd3e040f67515bb2411d03ccb27dcf29603fb699e1a450498381659764b9a8R107)), otherwise I will be mocking a lot of things